### ISSUE TITLE
404 Errors after Chapter 2

### ISSUE THREAD

**Issue by cdesign** [SymfonyCasts/api-platform#3]
Hello.. I am having difficulties with routing (apparently) at the end of Chapter 2. When I attempt to load 'http://mysite/api', I get a 404 error. The API Platform ui does not load. I know the site is working because I can load the Symfony welcome page as well as static html files. And all of the 'api' routes are registered. When I run `debug:router` I get the following:
```
 ------------------------------------- -------- -------- ------ ------------------------------------- 
  Name                                  Method   Scheme   Host   Path                                 
 ------------------------------------- -------- -------- ------ ------------------------------------- 
  _twig_error_test                      ANY      ANY      ANY    /_error/{code}.{_format}             
  api_entrypoint                        ANY      ANY      ANY    /api/{index}.{_format}               
  api_doc                               ANY      ANY      ANY    /api/docs.{_format}                  
  api_jsonld_context                    ANY      ANY      ANY    /api/contexts/{shortName}.{_format}  
  api_cheese_listings_get_collection    GET      ANY      ANY    /api/cheese_listings.{_format}       
  api_cheese_listings_post_collection   POST     ANY      ANY    /api/cheese_listings.{_format}       
  api_cheese_listings_get_item          GET      ANY      ANY    /api/cheese_listings/{id}.{_format}  
  api_cheese_listings_delete_item       DELETE   ANY      ANY    /api/cheese_listings/{id}.{_format}  
  api_cheese_listings_put_item          PUT      ANY      ANY    /api/cheese_listings/{id}.{_format}  
 ------------------------------------- -------- -------- ------ -------------------------------------
```
But all routes generate a 404 error when sent via Postman.

Now I did discover that if I set the 'api_platform: prefix' (in config/routes/api_platform.yaml') to just a slash '/', I can all of a sudden load the API Platform ui by going to the index page (i.e. 'http://mysite'). But.. when I then use the 'Try it out' links to send a request, I again get a 404 error.

So there appears to be some configuration issue that is incorrect. Or.. it also sounds similar to this old bug:

<a href="https://github.com/api-platform/api-platform/issues/584">API Platform Issue 584</a>
<a href="https://github.com/api-platform/admin/issues/118">API Platform/Admin Issue 118</a>

I should state that I started the tutorial by installing a fresh copy of Symfony <b>4.3</b> skeleton, and then I installed API Pack and maker via recipes. So the only difference with my source and the tutorial is that i am using Symfony 4.3 rather than 4.2.

Any idea what might be causing this issue? or how best to debug it?

Thx!

**Comment by cdesign** [comment]
I figured out the issue. I am using MAMP Pro (i.e. Apache, not the built-in php web server). I forgot that Symfony 4 no longer installs a .htaccess file. Without those rewrite rules, the front controller was never invoked for any route other than 'http://mysite'.

To fix the issue, I ran: `composer require symfony/apache-pack` as mentioned [here](https://symfony.com/doc/current/setup/web_server_configuration.html#adding-rewrite-rules).
